sql-plugin/src/main/scala/com/nvidia/spark/rapids/higherOrderFunctions.scala

```diff
@@ -278,7 +278,8 @@ trait GpuArrayTransformBase extends GpuSimpleHigherOrderFunction {
    * Post-process the column view of the array after applying the function parameter
    */
   protected def transformListColumnView(
-    lambdaTransformedCV: cudf.ColumnView): GpuColumnVector
+    lambdaTransformedCV: cudf.ColumnView,
+    arg: cudf.ColumnView): GpuColumnVector
```

Comment:
would be nice to add the definition of `arg` in this method header, it's not as clear to me as `lambdaTransformedCV`